suite/tests/api/dis-a64.txt

```diff
@@ -52818,3 +52818,4 @@ ce81fc1f : xar v31.2d, v0.2d, v1.2d, #0x3f           : xar    %q0 %q1 $0x3f $0x0
 4ec9790a : zip2 v10.2d, v8.2d, v9.2d                   : zip2   %q8 %q9 $0x03 -> %q10
 4ed37a54 : zip2 v20.2d, v18.2d, v19.2d                 : zip2   %q18 %q19 $0x03 -> %q20
 4edd7b9e : zip2 v30.2d, v28.2d, v29.2d                 : zip2   %q28 %q29 $0x03 -> %q30
+
```

Comment:
Superfluous blank line?

Reply:
The trailing blank line was added by the `./suite/tests/api/sort-dis-a64.py` script. I have updated to script to stop doing that